[2024-04-16T10:33:25.945+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-16T10:33:25.993+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: trigger_airbyte_job_example.airbyte_check_sync manual__2024-04-16T10:33:06.183521+00:00 [queued]>
[2024-04-16T10:33:26.014+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: trigger_airbyte_job_example.airbyte_check_sync manual__2024-04-16T10:33:06.183521+00:00 [queued]>
[2024-04-16T10:33:26.015+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-04-16T10:33:26.045+0000] {taskinstance.py:2327} INFO - Executing <Task(AirbyteJobSensor): airbyte_check_sync> on 2024-04-16 10:33:06.183521+00:00
[2024-04-16T10:33:26.059+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1486) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-04-16T10:33:26.062+0000] {standard_task_runner.py:63} INFO - Started process 1488 to run task
[2024-04-16T10:33:26.062+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'trigger_airbyte_job_example', 'airbyte_check_sync', 'manual__2024-04-16T10:33:06.183521+00:00', '--job-id', '57', '--raw', '--subdir', 'DAGS_FOLDER/dag_aribyte_ex.py', '--cfg-path', '/tmp/tmpfzb6o8_r']
[2024-04-16T10:33:26.068+0000] {standard_task_runner.py:91} INFO - Job 57: Subtask airbyte_check_sync
[2024-04-16T10:33:26.202+0000] {task_command.py:426} INFO - Running <TaskInstance: trigger_airbyte_job_example.airbyte_check_sync manual__2024-04-16T10:33:06.183521+00:00 [running]> on host ff62e46d2b31
[2024-04-16T10:33:26.581+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='trigger_airbyte_job_example' AIRFLOW_CTX_TASK_ID='airbyte_check_sync' AIRFLOW_CTX_EXECUTION_DATE='2024-04-16T10:33:06.183521+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-16T10:33:06.183521+00:00'
[2024-04-16T10:33:26.587+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-16T10:33:26.588+0000] {baseoperator.py:399} WARNING - AirbyteJobSensor.execute cannot be called outside TaskInstance!
[2024-04-16T10:33:26.610+0000] {base.py:84} INFO - Using connection ID '***_call_airbyte' for task execution.
[2024-04-16T10:33:26.690+0000] {airbyte.py:104} INFO - Waiting for job 180 to complete.
[2024-04-16T10:34:26.700+0000] {base.py:84} INFO - Using connection ID '***_call_airbyte' for task execution.
[2024-04-16T10:34:26.762+0000] {airbyte.py:104} INFO - Waiting for job 180 to complete.
[2024-04-16T10:35:26.771+0000] {base.py:84} INFO - Using connection ID '***_call_airbyte' for task execution.
[2024-04-16T10:35:26.795+0000] {airbyte.py:99} INFO - Job 180 completed successfully.
[2024-04-16T10:35:26.796+0000] {base.py:294} INFO - Success criteria met. Exiting.
[2024-04-16T10:35:26.796+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-16T10:35:26.809+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=trigger_airbyte_job_example, task_id=airbyte_check_sync, execution_date=20240416T103306, start_date=20240416T103325, end_date=20240416T103526
[2024-04-16T10:35:26.840+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-16T10:35:26.867+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-16T10:35:26.872+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
